src/urbit/app/chess.hoon

```diff
@@ -662,6 +664,10 @@
                 :*  %give  %fact  ~[/active-games]
```

Comment:
See primary comment for this review. Line `421` should now be `%challenge-resolved`.